image/correction/tests/test_correction.py

```diff
@@ -0,0 +1,53 @@
+import unittest
```

Comment:
Always good to add a header block. Block would include authorship, license, copyright, dates and description of what the driver does.